aggregate_test.go

```diff
@@ -255,7 +255,7 @@ func Test_Aggregate(t *testing.T) {
 		// Test 4 ensures depending version bundles are not aggregated together in
 		// case their dependency definitions do not meet the defined constraints.
 		// Thus the aggregated state should be empty because there is no proper
-		// distribution available.
+		// release available.
```

Comment:
The concept is now called `release`. We rename this internally to stay aligned. See https://github.com/giantswarm/api-spec/pull/54.